templates/idx.html

```diff
@@ -60,12 +60,10 @@
 
             <div class="content">
                 <p>
-                    This site allows you to run a test WebAssembly workload in an Enarx Keep.
-                    Once you have logged in, you simply need to upload a WebAssembly file along with <a
-                        href="https://enarx.dev/docs/running/enarx_toml">the Enarx runtime configuration</a>.
```

Comment:
We should probably keep a link to the Enarx.toml docs somewhere

Reply:
You are probably right, let me fix that.